shuttle/db/db_functions.py

```diff
@@ -141,14 +141,18 @@ def commit_schedule(table, all_locations):
 
 def commit_shuttle_request(location):
     try:
+        username = flask_session['USERNAME']
+        sql = "SELECT * FROM SHUTTLE_REQUEST_LOGS WHERE ACTIVE = '{0}'".format('Y')
```

Comment:
You should just hardcode 'Y' in since it isn't a variable, just a char that will never change.